fedimint-api/src/encoding/mod.rs

```diff
@@ -131,6 +137,7 @@ impl_encode_decode_num!(u64);
 impl_encode_decode_num!(u32);
 impl_encode_decode_num!(u16);
 impl_encode_decode_num!(u8);
+impl_encode_decode_num!(usize);
```

Comment:
Instead of `impl`ing encode for `usize` we should rather change the type where it was needed to `u64`, otherwise encodings will depend on the bit width of the target platform (so definitely not good for *consensus* encoding).